Andrii_Voinenko/Lesson9/task1.py

```diff
@@ -0,0 +1,24 @@
+from typing import Callable
+
+
+def first_decorator(func: Callable):
+    def to_print_func(number1, number2):
+        return f'{str(func)}\n{func(number1, number2)}'
```

Comment:
not exactly what I want. 
<function multiplication at 0x7ff4a6d29790> it is not a function name. Try to find better solution